tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -1253,13 +1254,19 @@ def _fetch_new_requests(self) -> List[RequestQueueItem]:
+        try:
+            if self.dist.rank == 0:
+                # Preserve original `new_requests` on rank 0 since it may contain
+                # Python-only objects (e.g., custom logits processors) not serializable by pybind.
+                _ = self._broadcast_new_requests(new_requests,
+                                                 py_request_objects)
+            else:
+                new_requests, py_request_objects = self._broadcast_new_requests(
+                    new_requests, py_request_objects)
+        except Exception as e:
+            raise ValueError(
+                f"broadcast new requests failed: {str(e)}, payload size: {len(pickle.dumps((new_requests, py_request_objects)))} bytes"
```

Ruff (0.12.2)

Line too long (135 > 120)

(E501)